extensions/interactions/LogicProof/static/js/studentSpec.ts

```diff
@@ -16,7 +16,7 @@
  * @fileoverview Unit tests for LogicProof interaction student components.
  */
 
-errorWrapper = function(
+var errorWrapper = function(
```

Comment:
Ditto.

Also this is possibly a good opportunity to use the new es6 spread operator.

Reply:
This cannot be moved out of the global scope since it is used in multiple describe blocks. A rename for errorWrapper in sharedSpec works fine here and so the es6 operator is not required.